packages/protocol/migrationsConfig.js

```diff
@@ -84,14 +85,14 @@ const DefaultConfig = {
     frozen: true,
   },
   governance: {
-    queueExpiry: WEEK, // Change to 4 weeks once mainnet activated
-    dequeueFrequency: MINUTE, // Change to 1 week once mainnet activated
-    concurrentProposals: 3, // Change to 10 once mainnet activated
-    approvalStageDuration: 30 * MINUTE, // Change to 3 days once mainnet activated
-    referendumStageDuration: HOUR, // Change to 1 week once mainnet activated
-    executionStageDuration: WEEK,
+    queueExpiry: 4 * WEEK,
+    dequeueFrequency: DAY, // Change to 1 week once mainnet activated
+    concurrentProposals: 3,
+    approvalStageDuration: DAY, // Change to 3 days once mainnet activated
```

Comment:
one thing that we may consider is allowing the proposal to move forward immediately to referendum once approved by the required number of signers

Reply:
same with referendum -- once it hits the required number of votes and gold value just move it forward (unless we think people are likely to change/retract their votes).  though this won't work if you change the participateBaseline to 0

Reply:
> one thing that we may consider is allowing the proposal to move forward immediately to referendum once approved by the required number of signers

This was a deliberate choice in order to keep a predictable governance cadence

>  once it hits the required number of votes and gold value
What do you mean by this?

Reply:
i wonder how valuable that is to keep the approval stage for full length though.  you can still keep the cadence of how many get dequeued.  so the overall rate of approvals through the pipeline are still consistent.

> once it hits the required number of votes and gold value

I just mean if we hit a threshold of x/totalGold has voted yes, you can also accelerate to the next phase early

Reply:
> you can still keep the cadence of how many get dequeued
But you'd lose the cadence of when folks need to vote, no?